docs/Makefile

```diff
@@ -175,6 +178,8 @@ linkcheck:
 
 doctest:
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
+	rm -rf data/local/
```

Comment:
instead of putting this here, you can call `make doctest clean` in place of `make doctest`